core/controllers/contributor_dashboard_admin.py

```diff
@@ -256,3 +259,85 @@ def get(self):
                 'can_submit_questions': user_rights.can_submit_questions
             })
         self.render_json(response)
+
+
+class TranslationContributionStatsHandler(base.BaseHandler):
+    """Handler to show the translation contribution stats of a user."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'username': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
+        }
+    }
+
+    @acl_decorators.can_access_contributor_dashboard_admin_page
```

Comment:
Only the translation admins should be allowed to check translation-related stats. I've created issue #13365 to track this task!